plugins/modules/sda_fabric_devices_workflow_manager.py

```diff
@@ -4022,6 +4022,55 @@ def get_diff_merged(self, config):
 
         return self
 
+    def arrange_config_for_deletion(self, device_config):
+        """
+        Arrange the device config in such a way that the device with the role 'CONTROL_PLANE_NODE'
```

Comment:
```
    Prioritize the device config so that the device with the role 'CONTROL_PLANE_NODE'
    is deleted last, and devices that don't exist are handled first.
```